install/kubernetes/cilium/files/hubble/dashboards/hubble-dashboard.json

```diff
@@ -0,0 +1,3232 @@
+{
```

Comment:
This dashboard is not used by the Helm template. Is that intentional?

The metrics themselves are exposed by Cilium-Agent as well, but on a different port (which is intentional, since we expect the two metric ports to potentially be consumed by different teams)

Reply:
I thought https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/templates/hubble/dashboards-configmap.yaml#L2 was importing this as well.

Reply:
yeah, the ConfigMap for Hubble dashboards already exists and will include this one